udata_front/theme/gouvfr/templates/footer.html

```diff
@@ -125,56 +122,62 @@ <h3 class="h4">{{ _('Social networks') }}</h3>
-                    <a href="{{ url_for('site.home') }}" title="{{ _('Go back to home page') }}" class="mw-100">
-                        {% include theme('svg/marianne.svg') %}
-                    </a>
-                </div>
-                <ul class="fr-col-12 fr-col-lg-6 fr-grid-row network">
+    </div>
+    <div class="fr-container">
+        <div class="fr-footer__body">
+            <div class="fr-footer__brand fr-enlarge-link">
+                <a href="/" title="Retour à l’accueil">
+                    <p class="fr-logo" title="république française">
+                        république
+                        <br>française
+                    </p>
+                </a>
+            </div>
+            <div class="fr-footer__content">
+                <ul class="fr-footer__content-list">
                     {% for item in nav.gouvfr_network %}
-                    <li class="mr-lg mt-md fr-text--bold">
-                        <a class="unstyled" href="{{ item.url }}">{{ item.label }}</a>
+                    <li class="fr-footer__content-item">
+                        <a class="fr-footer__content-link" href="{{ item.url }}">{{ item.label }}</a>
                     </li>
                     {% endfor %}
                 </ul>
-                <div class="fr-col-6 fr-col-sm-4 fr-col-md-3 fr-col-lg-2 fr-ml-auto fr-mr-1v">
-                    <select
-                            aria-label="{{ _('Language selector') }}"
-                            class="fr-select"
-                            onchange="location = location.href.replace(new RegExp('\/\\w{2}\/'), '/' + this.value + '/', regex=true)"
-                        >
-                            {% for code, name in config['LANGUAGES'].items() %}
-                                {% if g.lang_code == code %}
-                                <option value={{code}} selected>
-                                    {{name}}
-                                </option>
-                                {% else %}
-                                <option value={{code}}>
-                                    {{name}}
-                                </option>
-                                {% endif %}
-                            {% endfor %}
-                        </select>
-                </div>
             </div>
-        </footer>
-        <footer class="footer-legals fr-py-3w text-grey-380 fr-text--sm fr-mb-0">
-            <div class="fr-grid-row fr-grid-row--gutters align-items-center">
-                <ul class="fr-col-12 fr-col-lg fr-grid-row legal-nav text-align-center-lg">
-                    {% for item in nav.gouvfr_footer %}
-                    <li>
-                        <a href="{{ '' if item.items else item.url }}" title="{{ item.label }}" class="unstyled">
+            <div class="fr-col-6 fr-col-sm-4 fr-col-md-3 fr-col-lg-2 fr-ml-auto self-end">
+                <select
+                        aria-label="{{ _('Language selector') }}"
```

Comment:
Saw that Language selector is in DSFR roadmap (expected in T2 2022, v1.7.0)

Reply:
Yes ! I don't know how this will work with more than 2 languages